client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -19,92 +19,90 @@ import getInlineHelpSearchResultsForQuery from 'state/inline-help/selectors/get-
 import getSelectedResultIndex from 'state/inline-help/selectors/get-selected-result-index';
 import isRequestingInlineHelpSearchResultsForQuery from 'state/inline-help/selectors/is-requesting-inline-help-search-results-for-query';
 import hasInlineHelpAPIResults from 'state/selectors/has-inline-help-api-results';
-import { getLastRouteAction } from 'state/ui/action-log/selectors';
-import { setSearchResults, selectResult } from 'state/inline-help/actions';
+import { selectResult } from 'state/inline-help/actions';
 import { localizeUrl } from 'lib/i18n-utils';
 
-class InlineHelpSearchResults extends Component {
-	static propTypes = {
-		openResult: PropTypes.func.isRequired,
-		translate: PropTypes.func,
-		searchQuery: PropTypes.string,
-	};
+function HelpSearchResults( {
+	hasAPIResults = false,
+	isSearching = false,
+	openResult,
+	searchQuery = '',
+	searchResults,
+	selectedResultIndex = -1,
+	selectSearchResult,
+	translate = identity,
+	placeholderLines,
+} ) {
+	const selectResultHandler = useCallback(
```

Comment:
The component where this is used is not being memoized. Therefore [`useCallback`  is probably an over-optimisation](https://aheadcreative.co.uk/articles/when-to-use-react-usecallback/) and we can remove it.

Reply:
You've fixed this now.